pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -310,15 +359,29 @@ func (t ResolvedPipelineTask) isScheduled() bool {
 	return len(t.TaskRuns) > 0
 }
 
-// haveAnyRunsFailed returns true when any of the taskRuns/customRuns have succeeded condition with status set to false
+// haveAnyRunsFailed returns true when any of the child (PinP) PipelineRuns/TaskRuns/CustomRuns have succeeded condition with status set to false
```

Comment:
NIT: I referred in the past to pipelines-in-pipelines as pip, which can be confused with the Python tool. 
PinP avoids that confusion, but it also sounds like a not-so-nice word. Not sure what the official acronym should be :P